src/halmos/config.py

```diff
@@ -109,18 +110,29 @@ def parse(values: str | None) -> dict[str, list[int]]:
         if not values:
             return {}
 
-        # TODO: update syntax: name1={size1,size2},name2=size3,...
+        # syntax: --array-lengths name1=sizes1,name2=sizes2,...
+        # where sizes is either a comma-separated list of integers enclosed in curly braces, or a single integer
+
+        # remove all white spaces to simplify the pattern matching
+        values = "".join(values.split())
```

Comment:
I don't understand this line, we don't expect whitespace in the middle of the values, right? why not just `strip()`?

Reply:
we do expect whitespaces, like: `halmos --array-lengths "x={1,   2}"`